### PR TITLE
[launcher] add animations to onboarding tutorial

### LINKED ISSUE #1483: Visual/Motion Design: Updates to onboarding slides
Preview: https://xd.adobe.com/view/e93def3b-fb95-4b67-42b2-963ccd7db58c-9f0c/
Specs: https://xd.adobe.com/spec/200f3ec0-0db7-461e-5135-2622568d0565-563e/
Animation mp4's: https://drive.google.com/drive/folders/18lHNfTnKJbSfBVzKx0X127fU_103JKpb?usp=sharing 

This update contains the motion design work for the onboarding slides we've done with @kyleFirethought. New update includes 4 slides with animation loops illustrating each slides key topics. 

- Each animation should be playing in a endless loop. No controls are displayed.
- The UI layout has been slightly adjusted, area for animations is now larger in width. 
- Preview includes an updated copy. This is largely based on the copywriting contributions done for Decred booklet. If anyone wishes to contribute in refining it for a smoother and more concentrated reading experience – post here or #1402 and discuss with #writers_room on slack. 
- Typographic concerns: the text styles are with Source Sans Pro Regular and Semi-Bold (not normal Bold). Lighter text in brackets is with color #959DAC. Bulletpoints in slide 3 should be styled accordingly outside the paragraph div. 

screen shot 2018-06-20 at 16 49 21

### LINKED ISSUE #1488: .mp4 file update: Motion Blur enhancement fix to scene 1
https://github.com/matheusd/decred-weekly-builds/releases/tag/v20180514165803

After reviewing the onboarding video content for all scenes have been updated with a few cosmetic enhancements, noteably light motion blur. **This effects all scenes 1-4**

There is also a slight issue with scene 1 not looping correctly,  this has been fixed in the most recent version available in the shared directory: https://drive.google.com/drive/folders/18lHNfTnKJbSfBVzKx0X127fU_103JKpb?usp=sharing

Other than that no other issues seen in the Dev Build.  Tested: Manjaro KDE Edition (17.1.10)

### PR DESCRIPTION
**Pull request by matheusd**
Close #1483 
Close #1488

### CODE REVIEW

**Comment by alexlyp** [comment]
Animations look good.  

Just a couple small nits: 
1) On Screen 4 (Safety Tips), the skip button disappears. I guess if that's now the last page it should change to finish or something instead.

2) The Next/Skip tool bar shifts downward when going to Screen 3 (Staking and Governance).  Maybe we can do wide colums to have less vertical space used.  Or have the Next/Skip stay at the same position at the bottom.  Thoughts?

File: app/style/Tutorial.less [hunk 1]
```diff
@@ -117,6 +64,22 @@
       margin: 0em 5em 2em 0em;
       break-inside: avoid-column;
     }
+
+    ul {
+      margin: 0em 5em 2em -3.1em;
```
Comment (alexlyp) [inline app/style/Tutorial.less]: margin-right should be 2em

for `<p>` too

Comment (alexlyp) [reply app/style/Tutorial.less]: That seems to space out the text better so it won't push down the Staking and Governance buttons

**Comment by alexlyp** [comment]
I was playing around a bit with the buttons too,  Skip is still spaced as though there were 5 steps.  Looks like `margin-left: 10px, margin-top: 8px` works well.  

`line-height` should be set to 19px for `.tutorial-main-text`

**Comment by matheusd** [comment]
Updated the copy and did the fixes. Ready for final review.

**Comment by matheusd** [comment]
Updated animations to close #1488

**Pull request merged**